rca/programmes/migrations/0051_update_revisions.py

```diff
@@ -0,0 +1,21 @@
+import json
+
+from django.core.serializers.json import DjangoJSONEncoder
+from django.db import migrations
+
+from rca.programmes.models import ProgrammePage
+
+
+def update_revisions_with_current_programme_type(apps, schema_editor):
+    for page in ProgrammePage.objects.all():
```

Comment:
Models in migrations need to be imported from the app registry, like:
`ProgrammePage = apps.get_model('programmes', 'ProgrammePage')`.

See https://docs.djangoproject.com/en/3.0/topics/migrations/#data-migrations